zerver/openapi/zulip.yaml

```diff
@@ -448,6 +448,206 @@ paths:
                           ],
                         "upload_space_used": 571946,
                       }
+  /drafts:
+    get:
+      operationId: fetch_drafts
+      tags: ["drafts"]
+      description: |
+        Fetch all drafts for the current user.
+
+        `GET {{ api_url }}/v1/messages`
+      responses:
+        "200":
+          description: Success.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/JsonSuccess"
+                  - properties:
+                      count:
+                        type: integer
+                        description: |
+                          The number of drafts the user currently has. Also the
+                          number of drafts returned under "drafts".
+                        example: 3
```

Comment:
Hmm, this `count` field feels unnecessary; the client can easily compute that.  Maybe better to just remove it?

Reply:
We could, and I agree that we don't 100% need it. But what if the client (not necessarily our web/mobile clients) didn't want to have to loop over all of the elements before figuring out how many drafts were returned? Is there any harm in keeping this?